doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -236,23 +276,23 @@ For the User OS, we are using the same `Clear Linux`_ release version as the Ser
   .. code-block:: none
 
      cd ~
-     wget https://download.clearlinux.org/releases/24040/clear/clear-24040-kvm.img.xz
-     unxz clear-24040-kvm.img.xz
+     wget https://download.clearlinux.org/releases/26440/clear/clear-26440-kvm.img.xz
+     unxz clear-26440-kvm.img.xz
 
 * Download the Production Kernel (PK) kernel
 
   .. code-block:: none
 
-     wget https://download.clearlinux.org/releases/24040/clear/x86_64/os/Packages/linux-pk414-standard-4.14.57-69.x86_64.rpm
-     rpm2cpio linux-pk414-standard-4.14.57-69.x86_64.rpm | cpio -idmv
+     wget https://download.clearlinux.org/releases/26440/clear/x86_64/os/Packages/linux-iot-lts2018-4.19.0-22.x86_64.rpm
+     rpm2cpio linux-iot-lts2018-4.19.0-22.x86_64.rpm | cpio -idmv
 
 * Update the UOS kernel modules
 
   .. code-block:: none
 
-     sudo losetup -f -P --show /root/clear-24040-kvm.img
+     sudo losetup -f -P --show /root/clear-26440-kvm.img
```

Comment:
```suggestion
     sudo losetup -f -P --show ~/clear-26440-kvm.img
```

This is to account for the fact that the user is possibly (and most likely) not root (since we originally download the image in `~/`